src/NuGet.Core/NuGet.Commands/TrustedSignersCommand/TrustedSignerActionsProvider.cs

```diff
@@ -58,7 +58,7 @@ public async Task SyncTrustedRepositoryAsync(string name, CancellationToken toke
 
                     _trustedSignersProvider.AddOrUpdateTrustedSigner(existingRepository);
 
-                    await _logger.LogAsync(LogLevel.Information, string.Format(CultureInfo.CurrentCulture, Strings.SuccessfullySynchronizedTrustedRepository, name));
+                    await _logger.LogAsync(LogLevel.Minimal, string.Format(CultureInfo.CurrentCulture, Strings.SuccessfullySynchronizedTrustedRepository, name));
```

Comment:
I get this idea from @kartheekp-ms 's recent [PR#3992](https://github.com/NuGet/NuGet.Client/pull/3992) 
Without this change `dotnet nuget trust` command show nothing on cli with successful action.  Only show something if there is error/warning. 
With this change we can see result exactly same as NuGet.exe for successful action. I believe there is no impact on `NuGet.exe`.

Reply:
@kartheekp-ms  Also can comment on this.

Reply:
> I believe there is no impact on NuGet.exe.

Isn't it the same code path for both dotnet and NuGet.exe?  How could the change affect only dotnet and not NuGet.exe?  I'm trying to understand.

Reply:
@dtivel @kartheekp-ms 
If no verbosity is passed then verbosity default to LogLevel.Minimal:
https://github.com/NuGet/NuGet.Client/blob/5138176d007034f96afce7132f034403a3888c44/src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/XPlatUtility.cs#L42-L43

But this log is logged with `LogLevel.Information` wouldn't displayed due to below level check:
https://github.com/NuGet/NuGet.Client/blob/5138176d007034f96afce7132f034403a3888c44/src/NuGet.Core/NuGet.Common/Logging/LoggerBase.cs#L81-L84

After thinking how about instead of making change here in `TrustedSignerActionsProvider.cs`, how about I change below?
https://github.com/NuGet/NuGet.Client/blob/82ae3f5c20b2dec55e382539d258a68077f1c7d4/src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Signing/TrustedSignersCommand.cs#L115

To:
```
                    // If not verbosity passed then it defaults LogLevel.Minimal which result no message displayed because successfull message are logged at Loglevel.Information.
                    // Since NuGet LogLevel.Information is equal to MSBuild normal, we need to defaults "N"
                    setLogLevel(XPlatUtility.MSBuildVerbosityToNuGetLogLevel(verbosity.Value() ?? "N"));
```

Reply:
@kartheekp-ms
I just synched offline with @dtivel.
I'll leave code as it's for now, but I'll post table shows if there is any change of user experience in output.

Reply:
@dtivel  @kartheekp-ms 
I created comparison table between latest `nuget(v5.9)` and `nugetcandidate`(compiled from my code), `dotnet nuget trust`(compiled from my code). Please note `dotnet nuget trust` is bit different in handling error cases than `nuget.exe`, in this case still I assumed as same, but it's not objective of this table. You can press on `✔️` or `exception` to see actual output behind it.


Action | nuget59 | nugetCandidate | dotnet |  
-- | -- | -- | --  | --
nuget59   trusted-signer list | [✔️](https://gist.github.com/erdembayar/7920e2f67701aeed56fb475121389bf5) | [✔️](https://gist.github.com/erdembayar/0d468b97c333cea1d80664615b8ff8e8) | [✔️](https://gist.github.com/erdembayar/ee7d8cacce5d843452b71ddebd22738a) |  
 nuget59 trusted-signer list -verbosity q |  [✔️](https://gist.github.com/erdembayar/a02b9832f2e4c26ac30e4f32c8e578db) | [✔️](https://gist.github.com/erdembayar/a02b9832f2e4c26ac30e4f32c8e578db)  | [✔️](https://gist.github.com/erdembayar/a02b9832f2e4c26ac30e4f32c8e578db)  |  
  nuget59 trusted-signer list -verbosity m|  [exception](https://gist.github.com/erdembayar/32513d55762d58e34ad2ebe2f938e05b) | [exception](https://gist.github.com/erdembayar/c338388eda94ba4bc4575e6b4713afb7)  |  [✔️](https://gist.github.com/erdembayar/391a28cca508901e0c8f5eee86d7b2ee/) |  
 nuget59 trusted-signer list -verbosity n | [✔️](https://gist.github.com/erdembayar/6d9ad50dc3d998d0f1c50f8a43aac0d6)  | [✔️](https://gist.github.com/erdembayar/97e0e150a0690405ed71d2ce4f98fb20)  |  [✔️](https://gist.github.com/erdembayar/d356f386fb9ce695a6af6bf1e56b26cc) |  
nuget59 trusted-signer list -verbosity d  | [✔️](https://gist.github.com/erdembayar/1f76478e90cdf167adb0a51da604aa68)  |  [✔️](https://gist.github.com/erdembayar/e9de2a97429507d966a00872ce4f59c7) | [✔️](https://gist.github.com/erdembayar/83f3899e1cd8f3516b95732cccc41f99)  |  
nuget59 trusted-signer list -verbosity diag  | [exception](https://gist.github.com/erdembayar/08d64ea5ec32c2b9cb748fa1a450b0ca)  | [exception](https://gist.github.com/erdembayar/ab209351b2b1ae2b199d3dd9946bbbca)  | [✔️](https://gist.github.com/erdembayar/e7807b3adafd3ddbd356bff83eb60fdc)  |  
nuget59 trusted-signers Add -Name NuGet -config ..\nuget.config  |  [✔️](https://gist.github.com/erdembayar/1959a6a54cbf1506f5b175160b5bd682) | [✔️](https://gist.github.com/erdembayar/f32c10b4b18a3e5cf3757aac0c43fdef)  | [✔️](https://gist.github.com/erdembayar/3ce5d1b2fe45655723fe512883d8bb81) |  
nuget59 trusted-signers add .\newtonsoft.json.13.0.1-beta1.nupkg -Name NugetOrg -Repository -config ..\nuget.config  |  [✔️](https://gist.github.com/erdembayar/6c48a0982292bdc154f8fa8677fd6e84) | [✔️](https://gist.github.com/erdembayar/bfd4070898f540200049c3669d2560d8)  | [✔️](https://gist.github.com/erdembayar/a6b28f763376a88ef6a62980114a6348)  |  
nuget59 trusted-signers Add -Name MyCompanyCert -CertificateFingerprint CE40881FF5F0AD3E58965DA20A9F571EF1651A56933748E1BF1C99E537C4E039 -FingerprintAlgorithm SHA256 -config ..\nuget.config  | [✔️](https://gist.github.com/erdembayar/ca2e878fb2dad550b226cf96a27aa644)  | [✔️](https://gist.github.com/erdembayar/fddfa71d82d34f547465ce9e0257db23)  | [✔️](https://gist.github.com/erdembayar/0fe317d8704cb981301c0a98fb4ec5c6)  |  
nuget59 trusted-signers Add -Name NuGetTrust -ServiceIndex https://api.nuget.org/v3/index.json -config ..\nuget.config -Owners "NugetTestOrg;MSTestOrg;TrustedTesOwners" -config ..\nuget.config  |  [✔️](https://gist.github.com/erdembayar/39e60e872ec33f4fea66cf63af4e29c9) | [✔️](https://gist.github.com/erdembayar/855c4fa782a000fc2d5932c6a9572878)  | [✔️](https://gist.github.com/erdembayar/e7905b4cc00c7c97171199a2923c48ae)  |  
nuget59 trusted-signers sync -Name NuGetTrust -config ..\nuget.config  |  [✔️](https://gist.github.com/erdembayar/a3d9f0f46d55f7b06efcbcdee90a326e) | [✔️](https://gist.github.com/erdembayar/a3d9f0f46d55f7b06efcbcdee90a326e)  | [✔️](https://gist.github.com/erdembayar/a3d9f0f46d55f7b06efcbcdee90a326e)  |  
nuget59 trusted-signers Remove -Name NugetTrust -config ..\nuget.config  |  [✔️](https://gist.github.com/erdembayar/66216e4f4dcb83156b8dac0bb2f962ca) |  [✔️](https://gist.github.com/erdembayar/66216e4f4dcb83156b8dac0bb2f962ca) | [✔️](https://gist.github.com/erdembayar/66216e4f4dcb83156b8dac0bb2f962ca) |  

Reply:
@dtivel  @kartheekp-ms 
I created comparison table between latest `nuget(v5.9)` and `nugetcandidate`(compiled from my code), `dotnet nuget trust`(compiled from my code). Please note `dotnet nuget trust` is bit different in handling error cases than `nuget.exe`, in this table I still assumed as same, but `dotnet nuget` trust it's not objective of this table, it's created to make sure `nuget(v5.9)` and `nugetcandidate` experience are still same . You can press on `✔️` or `exception` to see actual output behind it.


Action | nuget59 | nugetCandidate | dotnet |  
-- | -- | -- | -- | --
nuget59   trusted-signer list | [✔️](https://gist.github.com/erdembayar/7920e2f67701aeed56fb475121389bf5) | [✔️](https://gist.github.com/erdembayar/0d468b97c333cea1d80664615b8ff8e8) | [✔️](https://gist.github.com/erdembayar/ee7d8cacce5d843452b71ddebd22738a) |  
 nuget59 trusted-signer list -verbosity q |  [✔️](https://gist.github.com/erdembayar/a02b9832f2e4c26ac30e4f32c8e578db) | [✔️](https://gist.github.com/erdembayar/a02b9832f2e4c26ac30e4f32c8e578db)  | [✔️](https://gist.github.com/erdembayar/a02b9832f2e4c26ac30e4f32c8e578db)  |  
  nuget59 trusted-signer list -verbosity m|  [exception](https://gist.github.com/erdembayar/32513d55762d58e34ad2ebe2f938e05b) | [exception](https://gist.github.com/erdembayar/c338388eda94ba4bc4575e6b4713afb7)  |  [✔️](https://gist.github.com/erdembayar/391a28cca508901e0c8f5eee86d7b2ee/) |  
 nuget59 trusted-signer list -verbosity n | [✔️](https://gist.github.com/erdembayar/6d9ad50dc3d998d0f1c50f8a43aac0d6)  | [✔️](https://gist.github.com/erdembayar/97e0e150a0690405ed71d2ce4f98fb20)  |  [✔️](https://gist.github.com/erdembayar/d356f386fb9ce695a6af6bf1e56b26cc) |  
nuget59 trusted-signer list -verbosity d  | [✔️](https://gist.github.com/erdembayar/1f76478e90cdf167adb0a51da604aa68)  |  [✔️](https://gist.github.com/erdembayar/e9de2a97429507d966a00872ce4f59c7) | [✔️](https://gist.github.com/erdembayar/83f3899e1cd8f3516b95732cccc41f99)  |  
nuget59 trusted-signer list -verbosity diag  | [exception](https://gist.github.com/erdembayar/08d64ea5ec32c2b9cb748fa1a450b0ca)  | [exception](https://gist.github.com/erdembayar/ab209351b2b1ae2b199d3dd9946bbbca)  | [✔️](https://gist.github.com/erdembayar/e7807b3adafd3ddbd356bff83eb60fdc)  |  
nuget59 trusted-signers Add -Name NuGet -config ..\nuget.config  |  [✔️](https://gist.github.com/erdembayar/1959a6a54cbf1506f5b175160b5bd682) | [✔️](https://gist.github.com/erdembayar/f32c10b4b18a3e5cf3757aac0c43fdef)  | [✔️](https://gist.github.com/erdembayar/3ce5d1b2fe45655723fe512883d8bb81) |  
nuget59 trusted-signers add .\newtonsoft.json.13.0.1-beta1.nupkg -Name NugetOrg -Repository -config ..\nuget.config  |  [✔️](https://gist.github.com/erdembayar/6c48a0982292bdc154f8fa8677fd6e84) | [✔️](https://gist.github.com/erdembayar/bfd4070898f540200049c3669d2560d8)  | [✔️](https://gist.github.com/erdembayar/a6b28f763376a88ef6a62980114a6348)  |  
nuget59 trusted-signers Add -Name MyCompanyCert -CertificateFingerprint CE40881FF5F0AD3E58965DA20A9F571EF1651A56933748E1BF1C99E537C4E039 -FingerprintAlgorithm SHA256 -config ..\nuget.config  | [✔️](https://gist.github.com/erdembayar/ca2e878fb2dad550b226cf96a27aa644)  | [✔️](https://gist.github.com/erdembayar/fddfa71d82d34f547465ce9e0257db23)  | [✔️](https://gist.github.com/erdembayar/0fe317d8704cb981301c0a98fb4ec5c6)  |  
nuget59 trusted-signers Add -Name NuGetTrust -ServiceIndex https://api.nuget.org/v3/index.json -config ..\nuget.config -Owners "NugetTestOrg;MSTestOrg;TrustedTesOwners" -config ..\nuget.config  |  [✔️](https://gist.github.com/erdembayar/39e60e872ec33f4fea66cf63af4e29c9) | [✔️](https://gist.github.com/erdembayar/855c4fa782a000fc2d5932c6a9572878)  | [✔️](https://gist.github.com/erdembayar/e7905b4cc00c7c97171199a2923c48ae)  |  
nuget59 trusted-signers sync -Name NuGetTrust -config ..\nuget.config  |  [✔️](https://gist.github.com/erdembayar/a3d9f0f46d55f7b06efcbcdee90a326e) | [✔️](https://gist.github.com/erdembayar/a3d9f0f46d55f7b06efcbcdee90a326e)  | [✔️](https://gist.github.com/erdembayar/a3d9f0f46d55f7b06efcbcdee90a326e)  |  
nuget59 trusted-signers Remove -Name NugetTrust -config ..\nuget.config  |  [✔️](https://gist.github.com/erdembayar/66216e4f4dcb83156b8dac0bb2f962ca) |  [✔️](https://gist.github.com/erdembayar/66216e4f4dcb83156b8dac0bb2f962ca) | [✔️](https://gist.github.com/erdembayar/66216e4f4dcb83156b8dac0bb2f962ca) |  